pymc/find_optim_prior.py

```diff
@@ -0,0 +1,96 @@
+#   Copyright 2021 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from typing import List
+
+import aesara
+import aesara.tensor as aet
+import numpy as np
+
+from scipy import optimize
+
+import pymc as pm
+
+__all__ = ["find_optim_prior"]
+
+
+def find_optim_prior(
+    pm_dist: pm.Distribution,
+    lower: float,
+    upper: float,
+    init_guess: List[float],
```

Comment:
I think this should be a dictionary, sometimes order of dist parameters is not the most intuitive, and some distributions do require keyword arguments because of alternative parametrizations.

Also then you can return a dictionary that you use to initialize the distribution with `pm.Normal(name, **results)`

Reply:
Good point, will do